packages/flutter/lib/src/widgets/ticker_provider.dart

```diff
@@ -62,30 +62,87 @@ class TickerMode extends StatelessWidget {
     return widget?.enabled ?? true;
   }
 
+  /// Obtains a [ValueNotifier] from the [TickerMode] surrounding the `context`,
+  /// which indicates whether tickers are enabled in the given subtree.
+  ///
+  /// When tickers are enabled or disabled, the notifier notifies its listeners.
+  ///
+  /// While the [ValueNotifier] is stable for the lifetime of the surrounding
+  /// [TickerMode], calling this method does not establish a dependency between
+  /// the `context` and the [TickerMode]. When the [Widget] owning the `context`
```

Comment:
Explain the difference between establishing a dependency or not and why you'd want to avoid it (unnecessarily rebuilding to stop an animation controller, building anyway once the animation controller is started)

Reply:
Re-wrote this section to incorporate that information.